aqua/qt5/Portfile

```diff
@@ -1643,6 +1643,17 @@ foreach {module module_info} [array get modules] {
                     patchfiles-append patch-qtwebkit-MachineStackMarker.cpp.diff
                 }
 
+                # Compiles on arm64 and intel occasionally fails due to similar header files from the system/xcode/other
+                # macports headers being found on the include search path
+                # Resolve this by making sure that the qt5-qtwebkit headers are first on the include path
+                # https://trac.macports.org/ticket/63877
+                configure.cppflags-delete "-I${prefix}/include"
+                configure.cppflags-append "-I${worksrcpath}/include:-I${prefix}/include"
```

Comment:
I don't think I've ever seen a colon used in this position before... does that work?

```clang++ -I/path/to/something:-I/opt/local/include test.cpp```

Reply:
Nope - that is a mistake.  And unfortunately, the compile only worked due to the error.  Substituting in a space for the colon does not build